org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/info/InfoWindow.java

```diff
@@ -2624,7 +2624,25 @@ protected void enableExportButton()
 		
 		exportButton.setEnabled(contentPanel.getRowCount() > 0);		
 	}
-	
+
+	/** Return (if exists) the editor with the name */
+	public WEditor getEditor(String name) {
+		for (WEditor editor : editors) {
+			if (editor.getGridField() != null && editor.getGridField().getColumnName().equals(name))
+				return editor;
+		}
+		return null;
+	}
+
+	/** Return (if exists) the index of the column in the grid */
+	public int getColumnIdx(String columnName) {
+		for (int i = 0; i < contentPanel.getColumnCount(); i++) {
+			if (p_layout[i].getGridField() != null && p_layout[i].getGridField().getColumnName().equals(columnName))
+				return i;
+		}
+		return -1;
+	}
+
```

Comment:
Please use standard javadoc notation (i.e `@param` and `@return`).  Also, the parameter name for getEditor should be columnName instead of name.

Reply:
done